packages/capabilities/src/voucher.js

```diff
@@ -31,7 +31,7 @@ export const Identity = URI.match({ protocol: 'mailto:' })
 /**
  * Services are identified using did:key identifier.
  */
-export const Service = DID.match({ method: 'key' })
+export const Service = DID.match({})
```

Comment:
```suggestion
export const Service = DID.match({ method: 'web' })
```

Should the above be ^^ instead ? Also comment need to be updated

Reply:
I intended the type to allow any did. specifically I wanted to at least allow both `did:key` and `did:web`.
Does it matter much right now? we can narrow the type later if needed.... it seems like at this capabilities layer it shouldn't care which did method?